static/app/types/index.tsx

```diff
@@ -103,6 +104,18 @@ declare global {
      * See sentry/js/ads.js for how this global is disabled.
      */
     adblockSuspected?: boolean;
+
+    //typing currently used for demo add on
+    //TODO: improve typing
+    SentryApp?: {
+      HookStore: any;
+      ConfigStore: any;
+      Modal: any;
```

Comment:
@billyvg think I need to type these exports coming from the file better? I could do something like `type of` with a require statement if that's better.

Reply:
Hmm if you're accessing these globals, it's probably outside of a TS environment, so maybe not that important?

Reply:
@billyvg It is in a TS environment but it becomes difficult to replicate the types and probably would have the same probability of breaking since the type in the demo JS repo would have to be updated as well. I agree there isn't too much value in typing it.